### PR TITLE
[1177] - Strip column names before adding in DB

### PR DESCRIPTION
**Pull request by muhsinkamil**
Fixes https://github.com/centerofci/mathesar/issues/1177

**Technical details**

The column names contained white spaces when adding new table. Hence on returning, the values were `undefined`. Removed the extra whitespaces before adding new table

**Screenshots**

Screenshot 2022-03-21 at 2 12 58 PM

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by kgodey — CHANGES_REQUESTED** [review]
@muhsinkamil This code looks fine to me, thanks for the contribution! Could you please add a test or two for the functionality?

**Comment by muhsinkamil** [comment]
Thank you @kgodey, I will look into it.

**Comment by muhsinkamil** [comment]
@kgodey Sorry, I was waiting for action to be complete. But it's been going on forever 😞

**Review by kgodey — APPROVED** [review]
Looks great @muhsinkamil! Please fix linting errors and this can be merged.

File: mathesar/tests/data/col_names_with_spaces.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+id,a ,b
+1,"foo","bar"
+2,"baz","bam"
```
Comment (kgodey) [inline mathesar/tests/data/col_names_with_spaces.csv:3]: Please add newline at the end of the file.

**Pull request merged**